### ISSUE TITLE
Add a publications menu in admin

### ISSUE THREAD

**Issue by libreliodev** [libreliodev/wordpress#48]
This should be similar to the publications menu we have in our javascript admin console (http://admin.librelio.com/publications.html).

**Comment by serverfire** [comment]
@libreliodev We can put librelio admin in a frame. And add an automatic login in javascript project.
Also we can change layout of librelio admin for wp users. 
What do you think?

**Comment by libreliodev** [comment]
@serverfire Good idea. But then we should not include the menus of the js admin, just individual pages. What do you think.

**Comment by serverfire** [comment]
@libreliodev Yes that's good.
